packages/server/src/helpers/sendPushNotification.ts

```diff
@@ -2,6 +2,12 @@ import { messaging as adminMessaging } from "firebase-admin";
 import { CollectionReference } from "@google-cloud/firestore";
 import { MemberId } from "@raha/api-shared/dist/models/identifiers";
 
+// Currently this is copied from raha-mobile//deeplinking.ts.
```

Comment:
Any plans to change that?

Reply:
Not right now. I'll remove the `currently` lol.